eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/configuration/BeanNames.java

```diff
@@ -2,7 +2,17 @@
 
 public class BeanNames {
 
-    public static final String EXCLUDE_VARIANTS_LISTENER = "EXCLUDE_VARIANTS_LISTENER";
+    public static final String ACCESSION_RELEASE_JOB = "ACCESSION_RELEASE_JOB";
+
+    public static final String CREATE_RELEASE_STEP = "CREATE_RELEASE_STEP";
```

Comment:
RELEASE_MAPPED_ACTIVE_VARIANTS_STEP? This seems more specific considering we will create reports for variants that are:

* Active, merged and deprecated
* Mapped and unmapped